lib/event_store.ex

```diff
@@ -527,9 +527,16 @@ defmodule EventStore do
         Snapshotter.delete_snapshot(conn, source_uuid, opts)
       end
 
+      @accepted_overrides [:created_at]
```

Comment:
Perhaps `created_at` could be `override_created_at` to make the intent clear?

Reply:
What do you think about `created_at_override`, just a little concerned that `override_created_at` might sound more like something that expects a `boolean`?

Reply:
I have renamed it `created_at_override`, but happy rename it to your suggestion if you prefer.